### PR TITLE
Add doc for google sheets integration

### PR DESCRIPTION
**Pull request by michaeldjeffrey**

### CODE REVIEW

File: docs/use-the-network/console/integrations/google-sheets [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+id: google-sheets
+hide_title: true
+sidebar_label: Google Sheets
+slug: /use-the-network/consolee/integrations/google-sheets
```
Comment (PharkMillups) [inline docs/use-the-network/console/integrations/google-sheets]:
```suggestion
slug: /use-the-network/console/integrations/google-sheets
```

**Comment by PharkMillups** [comment]
Looks like you need to add `use-the-network/console/integrations/google-sheets` to sidebarDocs.js as part of the Integrations section. 

https://github.com/helium/docs/blob/master/sidebarsDocs.js#L79

**Review by PharkMillups — DISMISSED** [review]
Left a few comments. Otherwise looks good.

File: docs/use-the-network/console/integrations/google-sheets.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+We recommend setting questions to "Short Answer".
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/console/integrations/integrations-google-sheets-createform.png"
+  )}
+/>
+
+
+## Make sure public
```
Comment (PharkMillups) [inline docs/use-the-network/console/integrations/google-sheets.mdx]:
```suggestion
## Make Form Public
```

File: docs/use-the-network/console/integrations/google-sheets.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  src={useBaseUrl(
+    "img/use-the-network/console/integrations/integrations-google-sheets-decoderbody.png"
+  )}
+/>
+
+With that done, we can click **Add Integration**.
+
+## Attach to Device
+
+Going to **Devices** on the left-hand menu, we can now [add our label](../labels)
```
Comment (PharkMillups) [inline docs/use-the-network/console/integrations/google-sheets.mdx]:
```suggestion
Going to **Devices** on the left-hand menu, we can now [add our label](../labels).
```

File: docs/use-the-network/console/integrations/google-sheets.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Google Sheets
+
+## Creating a Form
+
+[Google Forms](https://forms.google.com)
+
+Create a form.
+Put in your fields, we're going to an environment sensor as an example.
```
Comment (PharkMillups) [inline docs/use-the-network/console/integrations/google-sheets.mdx]: This sentence is a bit confusing. I think we're missing some words?

Comment (michaeldjeffrey) [reply docs/use-the-network/console/integrations/google-sheets.mdx]: Good catch, thank you. I've updated some other verbiage as well.

**Review by kent-williams — APPROVED** [review]

**Review by kent-williams — APPROVED** [review]

**Pull request merged**